asskey-pass.yml

```
@@ -0,0 +1,125 @@
+name: SBOM Achieve Token
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)